### PR TITLE
zh-translation: blog/2020/workload-entry/index.md

### PR DESCRIPTION
**Pull request by orangegzx**
[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/blog/2020/workload-entry/index.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+这将创建一个带有一组标签和地址的新 `WorkloadEntry`，以及使用 `WorkloadSelector` 来选择所有带有所需标签的端点的 `ServiceEntry`，在这种情况下包括为 VM 创建的 `WorkloadEntry`。
+
+{{< image width="75%"
+    link="./workload-entry-final.svg"
+    alt="Service Entries Pointing to Workload Entries"
+    caption="The Internal of Service Entries Pointing to Workload Entries"
+    >}}
+
+注意 `ServiceEntry` 可以同时引用 Pod 和 `WorkloadEntries`，使用相同的选择器。现在 Istio 可以对 VM 和 Pod 进行相同的处理，而不是将它们分开。
```
Comment (kebe7jun) [inline content/zh/blog/2020/workload-entry/index.md:97]:
```suggestion
注意 `ServiceEntry` 可以同时引用 Pod 和 `WorkloadEntry`，使用相同的选择器。现在 Istio 可以对 VM 和 Pod 进行相同的处理，而不是将它们分开。
```

**Review by kebe7jun — APPROVED** [review]

**Comment by Xunzhuo** [comment]
It seems like a few tasks failed and merge is blocked

**Pull request merged**